src/main/java/core/basesyntax/dao/ActivityDaoCsvImpl.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Activity;
+import core.basesyntax.model.TypeActivity;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ActivityDaoCsvImpl implements ActivityDaoCsv {
```

Comment:
- this is not DAO - DAO works with a database ((or mock database in our case)), not an input file, this is a reader from what I see
- reading logic is mixed with parsing logic which breaks the single responsibility principle - create separate classes for these functions

Reply:
ok